index.html

```diff
@@ -167,9 +167,14 @@ <h2>CaptureController Extensions</h2>
             We extend {{CaptureController}} with {{CaptureController/oncapturedmousechange}}, an
             {{EventHandler}} for events of type {{CapturedMouseEvent}}.
           </p>
-          <p>Events MUST NOT be fired more than 30 times per second.</p>
           <p>
-            Events MUST NOT be fired after the last clone of the original video track is stopped.
+            User agents MAY limit the frequency with which events are fired. This can be achieved by
+            briefly buffering events, then skipping the event for one state and reporting only the
```

Comment:
Sorry, I don't really get this sentence especially the "skipping the event for one state"? I mean it looks similar to what I implemented in the prototype but I don't understand the wording.

Reply:
Yes, as a replacement to "then skipping the event for one state and reporting only the latest one".